linux_os/guide/system/accounts/accounts-pam/display_login_attempts/rule.yml

```diff
@@ -26,9 +26,6 @@ rationale: |-
 severity: low
 
 identifiers:
-    cce@rhel7: CCE-27275-7
-    cce@rhel8: CCE-80788-3
-    cce@rhel9: CCE-83560-3
```

Comment:
Instead moving CCE identifiers around, use identifiers available from this pool: https://github.com/ComplianceAsCode/content/blob/master/shared/references/cce-redhat-avail.txt